water-features.mss

```diff
@@ -167,7 +161,6 @@
 #springs {
   [natural = 'spring'][zoom >= 14] {
     marker-file: url('symbols/spring.svg');
-    marker-placement: interior;
```

Comment:
There don't seem to be the required changes to the springs SQL

Reply:
Fixed.